util/gen_coverage_report.ts

```diff
@@ -251,7 +249,7 @@ const buildTotals = (coverage: Coverage) => {
       totalsByContentType.raidboss++;
       totals.overall.raidboss++;
     }
-    if (thisCoverage?.oopsy?.num ?? 0 > 0) {
+    if ((thisCoverage?.oopsy?.num ?? 0) > 0) {
```

Comment:
This logic was completely broken, prior to this change it evaluated to:
```ts
    if (thisCoverage?.oopsy?.num ?? (0 > 0)) {
```
See:
![image](https://user-images.githubusercontent.com/6119598/168175891-c904581e-7f8a-4dda-ab85-10c16a57dd7c.png)

Reply:
Oops!